x-pack/plugins/fleet/common/openapi/paths/epm@bulk.yaml

```diff
@@ -0,0 +1,21 @@
+post:
+  summary: Packages - Bulk install
+  tags: []
+  responses:
+    '200':
+      description: OK
+      content:
+        application/json:
+          schema:
+            type: object
+            items:
```

Comment:
I don't see items in example response here: https://petstore.swagger.io/?url=https://raw.githubusercontent.com/elastic/kibana/330634f8f6be33bf3ad494b17167fbf107628bd1/x-pack/plugins/fleet/common/openapi/bundled.json#/default/bulk-install-packages